website/docs/registry/providers/publishing.html.md

```diff
@@ -14,7 +14,23 @@ description: |-
 
 Providers published to the Terraform Registry are written and built in the same way as other Terraform Providers. For guidance on how to write a provider, see [Writing Custom Providers](/docs/extend/writing-custom-providers.html).
 
-The provider repository on GitHub must match the pattern `terraform-provider-{NAME}`, and the repository must be public.
+The provider repository on GitHub must match the pattern `terraform-provider-{NAME}`, and the repository must be public.  All Terraform providers must contain one of the following open source licenses:
```

Comment:
This looks good! If it was under another heading (h3 or h4) we could link directly to the license documentation from any errors we generate when a provider is publishing with an invalid license.

Reply:
good suggestion, I'll update with a distinct header